plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/contribution/migration/CybersecurityMigrationContribution.java

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *   
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.cybersecurity.ui.contribution.migration;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.xmi.XMLResource;
+import org.polarsys.capella.core.data.capellacore.EnumerationPropertyLiteral;
+import org.polarsys.capella.core.data.capellacore.EnumerationPropertyType;
+import org.polarsys.capella.core.data.capellamodeller.Project;
+import org.polarsys.capella.core.data.migration.context.MigrationContext;
+import org.polarsys.capella.core.data.migration.contribution.AbstractMigrationContribution;
+import org.polarsys.capella.cybersecurity.model.CybersecurityQueries;
+import org.polarsys.capella.cybersecurity.model.SecurityNeeds;
+import org.polarsys.capella.cybersecurity.model.Threat;
+import org.polarsys.capella.cybersecurity.model.activator.CybersecurityModelActivator;
+
+public class CybersecurityMigrationContribution extends AbstractMigrationContribution {
+  Map<String, String> savedValues = new HashMap<>();
+  Project project;
+
+  @Override
+  public void unaryMigrationExecute(EObject currentElement, MigrationContext context) {
+    super.unaryMigrationExecute(currentElement, context);
+    if (currentElement instanceof Project) {
```

Comment:
Does it mean that you always create this configuration folder even though the viewpoint is not activated on the project ? A check should be done here.

Reply:
Ok.

Reply:
I have checked that there are values that need migration.